src/sensitivities/scalar.jl

```diff
@@ -10,18 +10,25 @@ import Base.identity
 @inline ∇(::typeof(identity), ::Type{Arg{1}}, x::Real) = one(x)
 
 # Some derivatives are missing from DiffRules.
-# TODO: Make a PR for DiffRules.
-@define_diffrule Base.:\(x, y) = :(-$y / $x^2), :(1 / $x)
-@define_diffrule Base.:transpose(x) = :(1)
+@define_diffrule Base.:abs(x) = :($x > 0 ? one($x) : -one($x))
```

Comment:
Would it make sense to let `abs'(0) = 0`?

Reply:
That makes sense, though `0` can be represented exactly, which would hit the discontinuity.